docs/source/architecture.rst

```diff
@@ -367,6 +367,26 @@ with the validators. N1 is the primary node; N2, N3, and N4 are secondary nodes.
     :alt: PBFT messages passed during normal operation
 
 
+.. _log-pruning-label:
+
+Log Pruning in Normal Mode
+--------------------------
+
+Sawtooth PBFT does not implement a checkpointing procedure (garbage collection
+for log messages). Instead, each node cleans the log periodically during normal
+operation.
+
+Log size is controlled by a configurable setting, as determined by the on-chain
+setting ``sawtooth.consensus.pbft.max_log_size``. When a block is committed,
+each node compares the size of its log against the maximum size. If the log
+exceeds this value, Sawtooth PBFT uses these rules to prune the log:
+
+- Keep blocks and messages for that sequence number (for the block that
```

Comment:
"Keep blocks and messages that are for the sequence number of the block that was just committed or higher" sounds more understandable (to me at least)

Reply:
Alas, "higher" is too far away from "sequence number" to be totally clear.  I'll try to find wording that works for us both.